lib/tdf3/src/utils/zip-reader.ts

```diff
@@ -1,6 +1,6 @@
-import { Buffer } from 'buffer';
 import { Manifest } from '../models/index.js';
 import { Chunker } from './chunkers.js';
+import { readUInt32LE, readUInt16LE, copyUint8Arr } from './index.js';
```

Comment:
you can say `index.js` instead of `index.ts` here?

Reply:
Its like a bundler thing, file should be called js even when they are ts. I know, looks not very intuitive ))